exercises/timesTable13.js

```diff
@@ -8,10 +8,18 @@
  *   13 times 3 is 39
  *   13 times 4 is 52
  */
+//I couldnt figure out how to print out all the multiples under the parameter
+//Im assuming I need an array to store the multiples as they run through
+//I only got it to print the input given.
 function timesTable13(n) {
-  // This is your job. :)
+  for (i = 1; i <= 1000; i++) {
+    if (n % 13 === 0) {
```

Comment:
If you put the following inside the loop:

```javascript
console.log('i is:', i);
console.log('n is:, n);
```

I think you'd see right away what you'd want to put here.

You want `if (i % 13 === 0) { ... }`, because the value of `n` never changes.